test/lld/em_asm_pthread.wasm.out

```diff
@@ -146,27 +152,15 @@
      (i32.const 0)
      (i32.const 156)
     )
-    (block
-     (drop
-      (i32.const 1588)
-     )
-     (drop
-      (i32.const 0)
-     )
-     (drop
-      (i32.const 70)
-     )
+    (memory.init 2
```

Comment:
why did these change?

Reply:
Because this tests wasm-emscripten-finalize, and this tools no longer does the removal the em_js/em_asm data which means it no longer modifies this part of the code and just leaves it as is.

The `--post-emscripten` pass now does this part.